File: programmers/lv2/이진_변환_반복하기.py
def solution(s):
    answer = [0, 0]

    while s != '1':
        zero = s.count('0')
        s = s.replace('0','', zero)
        s = bin(len(s))[2:]
        answer[0] += 1
        answer[1] += zero

    return answer

File: programmers/lv2/이진_변환_반복하기.js
function solution(s) {
  let str = s;
  const answer = [0, 0];
  while (str !== '1') {
    answer[0] += 1;
    let strArr = str.split('').filter((x) => x != '0');
    answer[1] += str.length - strArr.length;
    str = strArr.length.toString(2);
  }
  return answer;
}
